src/components/FilterEditor.tsx

```diff
@@ -299,17 +304,22 @@ export default class FilterEditor extends React.Component<FilterEditorProps, Fil
           />
           {this.state.valueIsSimpleFilter &&
             <div className="maputnik-expr-infobox">
-              You&apos;ve entered a old style filter,{' '}
-              <button
-                onClick={this.makeFilter}
-                className="maputnik-expr-infobox__button"
-              >
-                switch to filter editor
-              </button>
+              <Trans t={t}>
```

Comment:
Is there a better solution to this? Can we split the relevant translation into 2 keys?

Reply:
I'd rather not split translations unless completely necessary -- you brought up the point of RTL, but not only that, depending on the language it might be more natural to have elements in a different order than English. I haven't committed these changes yet, but for example, here:

https://github.com/maplibre/maputnik/blob/35840409b8435bc7a68502a6402cce8219b6f9ed/src/components/InputJson.tsx#L167

The translation in Japanese would make more sense as `<kbd>ESC</kbd> ..text..`

Reply:
I would prefer to have it only when absolutely needed, and I'm not sure this is the case with this text and button.
See the docs about trans - they are implying that in most cases, it's not needed:
https://react.i18next.com/latest/trans-component